### DISCUSSION TITLE
Internet speed

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#976] [General]
Anyone knows how can I check the avarage speed of internet in different spots in my house?
Cheers

**Comment by hernessteve** [comment]
Check different mobile apps

**Comment by MirkosMangle** [comment]
You can use the Netspot heat map tool https://apps.apple.com/us/app/netspot-wifi-map-speed-test/id1490247223?ls=11 to check the average speed of the internet in different spots of your house. This tool allows you to create detailed heat maps based on your floorplan, providing insights into the signal strength and speed of your WiFi network in various areas
